src/libm-tester/CMakeLists.txt

```diff
@@ -1,9 +1,13 @@
+# TODO: Find way to avoid relative includes.
 link_directories(${CMAKE_BINARY_DIR}/lib)
+# For sleef.h
 include_directories(${CMAKE_BINARY_DIR}/include)
+# For rename headers
+include_directories(${CMAKE_CURRENT_BINARY_DIR}/../libm/include)
```

Comment:
The reason I am trying to avoid ${CMAKE_SOURCE_DIR} and ${CMAKE_BINARY_DIR} is because that would force the assumption that SLEEF is the root cmake project. If you were to integrate SLEEF with LLVM, our build would break because the CMAKE_SOURCE_DIR would be the LLVM one.